packages/main/src/plugin/image-registry.ts

```diff
@@ -718,7 +718,7 @@ export class ImageRegistry {
   }
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  getBestManifest(manifests: any[], wantedArch: string, wantedOs: string): any {
+  getBestManifest(manifests: any[], wantedArch: string, wantedOs: string): any | undefined {
```

Comment:
Should an error be throw instead of returning `undefined` ?

Reply:
it's already returning undefined today as a normal case, I just added it into the signature to make it clearer

Reply:
https://github.com/containers/podman-desktop/blob/e070a2a678fae3a22a4754e3d24ee66c862c6715/packages/main/src/plugin/image-registry.ts#L707-L717

Reply:
I know convention naming can be discussed, but shouldn't we name `findBestManifest` then?

Reply:
I agree on the naming, but it'll be a follow-up PR

Reply:
np